server/src/schema.js

```diff
@@ -0,0 +1,68 @@
+const {gql} = require('apollo-server');
+
+const typeDefs = gql`
+    type Query {
+        events(pageSize: Int, after: String): EventConnection!
+        event(id: ID!): Event
+
+        # user information about the user currently logged in
+        me: User
+        user(id: ID!): User
+    }
+
+    type Mutation {
+        authGoogle(accessToken: AuthInput!): AuthResponse
+        authFacebook(accessToken: AuthInput!): AuthResponse
+        logout: Boolean! # success, then True.  
+    }
+
+    type AuthResponse {
+        token: String
+        name: String
+    }
+    input AuthInput {
+        accessToken: String!
+    }
+    
+    type User {
+        id: ID!
+        firstName: String!
+        lastName: String!
+        email: String!
+        age: Int
+        sex: Sex
+        city: String
+        state: String
+        phone: String
+    }
+
+    type Event {
```

Comment:
Shouldn't an event have a time and location? Otherwise, how would people attend?

Reply:
I added the Schedule type. Thank you!